docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-114.adoc

```diff
@@ -0,0 +1,47 @@
+== Clarifai API Key detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|TBD
+
+|Checkov Check ID
+|CKV_SECRET_114
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+A Clarifai API Key is vital for accessing Clarifai's powerful AI and machine learning services, which provide advanced image and video recognition capabilities. This key is crucial for authenticating requests to the API and ensuring that only authorized clients can utilize the services. Exposure of this key can lead to unauthorized access, misuse of the API, and potential data leaks. Securely managing this key is essential to maintain the security and integrity of your applications. Regularly reviewing your security settings and practices is important to prevent unauthorized usage. For detailed security guidelines and best practices, refer to the latest https://docs.clarifai.com/clarifai-basics/authentication/personal-access-tokens/[Clarifai documentation and support] resources.
```

Comment:
The Clarifai API Key API Key serves as an essential credential for accessing Clarifai API Key's AI and machine learning models services, which provide advanced image and video recognition capabilities. This key acts as a security measure, guaranteeing that only authorized applications can interact with the API and utilize the services provided. Exposing the API key can lead to unauthorized access, misuse of the API, and potential data leaks. For this reason, it's crucial to manage your Clarifai API Key securely and regularly review your security practices to prevent any unauthorized usage. For comprehensive security guidelines and best practices regarding MonkeyLearn API Key management, refer to the latest documentation and support resources available at https://docs.clarifai.com/clarifai-basics/authentication/personal-access-tokens/[Clarifai documentation and support].